deepforest/preprocess.py

```diff
@@ -141,32 +141,37 @@ def save_crop(base_dir, image_name, index, crop):
 def split_raster(annotations_file,
                  path_to_raster=None,
                  numpy_image=None,
-                 base_dir=".",
+                 save_dir=".",
                  patch_size=400,
                  patch_overlap=0.05,
                  allow_empty=False,
-                 image_name=None):
+                 image_name=None,
+                 base_dir=None):
     """Divide a large tile into smaller arrays. Each crop will be saved to
     file.
 
     Args:
         numpy_image: a numpy object to be used as a raster, usually opened from rasterio.open.read()
         path_to_raster: (str): Path to a tile that can be read by rasterio on disk
-        annotations_file (str): Path to annotations file (with column names)
-            data in the format -> image_path, xmin, ymin, xmax, ymax, label
-        base_dir (str): Where to save the annotations and image
-            crops relative to current working dir
+        annotations_file (str or pd.DataFrame): A pandas dataframe or path to annotations csv file. In the format -> image_path, xmin, ymin, xmax, ymax, label
+        save_dir (str): Directory to save images
```

Comment:
Are these two parameters doing the same thing?

save_dir (str): Directory to save images
base_dir (str): Directory to save images

Reply:
Yes, we have changed the name of the argument, the prior naming and doc string was misnamed. save_dir is to replace base_dir and we add a deprecation.